pkg/cli/credential.go

```diff
@@ -0,0 +1,79 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"github.com/acorn-io/cmd"
+	"github.com/gptscript-ai/gptscript/pkg/config"
+	"github.com/gptscript-ai/gptscript/pkg/credentials"
+	"github.com/spf13/cobra"
+)
+
+type Credential struct {
+	root        *GPTScript
+	AllContexts bool `short:"A" usage:"List credentials for all contexts" local:"true"`
+}
+
+func NewCredential(root *GPTScript) *cobra.Command {
+	c := cmd.Command(&Credential{root: root}, cobra.Command{
```

Comment:
I've been using the Customize interface approach to change the default cobra.Command fields.  I feel like that is slightly less confusing pattern. I'm going to probably drop support for this style of passing it as a second arg.

https://github.com/gptscript-ai/gptscript/blob/main/pkg/cli/gptscript.go#L102

Reply:
Fixed